runtime/src/core/grain-module.js

```diff
@@ -59,6 +59,10 @@ export class GrainModule {
     return this.instantiated.exports;
   }
 
+  get isGrainModule() {
+    return !!this.exports["GRAIN$MAIN"]
```

Comment:
Is this really the JS-idiomatic way to do to-bool?

Reply:
Yes, actually. https://github.com/airbnb/javascript#coercion--booleans

JS is the wild west.